Adding quick start entry for OpenShift

bergerhoffer opened the pull request:
Adding quick start entry for OpenShift, as per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#quick-start

bergerhoffer commented:
Note to self: This will need an icon. Depends which gets merged first, this PR or #107.

bergerhoffer commented:
@redhat-documentation/ccs-style-council for review please

mportman12 reviewed (changes requested):
Left one comment, otherwise LGTM

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc:
```diff
@@ -437,6 +437,22 @@ _Build an Operator using the Operator SDK._
+==== quick start (noun)
+*Description*: There are two types of quick starts in OpenShift:
+
+* quick starts that are guided tutorials in the web console
```
Done!

stoobie commented on supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc:
```diff
@@ -437,6 +437,22 @@ _Build an Operator using the Operator SDK._
+*Description*: There are two types of quick starts in OpenShift:
+
+* Quick starts that are guided tutorials in the web console
+* Quick start templates that allow users to quickly get started creating a new application
```
s/allow/enable

ianf77 replied:
enable or permit, instead of allow.
to quickly get started... hmm.  ... to start creating new applications quickly?

bergerhoffer replied:
Thanks, updated to enable and reworded some!

stoobie commented on supplementary_style_guide/style_guidelines/general-formatting.adoc:
```diff
@@ -26,7 +26,7 @@ Red Hat Enterprise Linux 8 delivers a stable, secure, and consistent foundation
 Avoid contractions in product documentation to leave no ambiguity and to make it easier for translation and international audiences.
 
-If you are writing quickstart or other content that uses a more informal xref:#conversational-style[conversational style] (_fairly conversational_ or _more conversational_), you may use contractions. In this case, follow the guidance in _The IBM Style Guide_ on using contractions.
+If you are writing quick start or other content that uses a more informal xref:#conversational-style[conversational style] (_fairly conversational_ or _more conversational_), you may use contractions. In this case, follow the guidance in _The IBM Style Guide_ on using contractions.
```
s/may/can

Avoid using 'may", to avoid ambiguity. Use "might" or "can". Unless you mean the month before June.

If you mean "Is permissible", then say that.

bergerhoffer replied:
@stoobie My update here is just changing "quickstart" to "quick start", not touching anything else in the style entry.

Additionally, this use of "may" was discussed in the original PR and it was determined to be an appropriate use of "may" per the ISG because it does mean permission.

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc:
```diff
@@ -437,6 +437,22 @@ _Build an Operator using the Operator SDK._
+==== quick start (noun)
+*Description*: There are two types of quick starts in OpenShift:
+
+* Quick starts that are guided tutorials in the web console
```
@dfitzmau I can't do a are/for swap here, so I tried to reword along the lines of how the next one is defined. Let me know if that's any better, or if you had a better suggestion for what you meant. Thanks!

```
Quick starts that provide a guided tutorial in the web console
```

dfitzmau replied:
HI @bergerhoffer . Thanks, LGTM.

ndeevy commented on supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc:
```diff
@@ -436,6 +436,22 @@ _Build an Operator using the Operator SDK._
+* Quick starts that provide a guided tutorial in the web console
+* Quick start templates that enable users to start creating new applications quickly
+
+Be sure to provide context about which type of quick start you are referring to.
```
```suggestion
Ensure that you provide context about which type of quick start you are referring to.
```
Suggestion to switch the phrasal verb out here. Feel free to ignore :)

ndeevy commented on supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc:
```diff
@@ -436,6 +436,22 @@ _Build an Operator using the Operator SDK._
+==== quick start (noun)
+*Description*: Two types of quick starts exist in OpenShift:
+
+* Quick starts that provide a guided tutorial in the web console
```
```suggestion
* Quick starts that provide a guided tutorial in the web console.
```

Complete sentence so suggesting a period.

ndeevy commented on supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc:
```diff
@@ -436,6 +436,22 @@ _Build an Operator using the Operator SDK._
+*Description*: Two types of quick starts exist in OpenShift:
+
+* Quick starts that provide a guided tutorial in the web console
+* Quick start templates that enable users to start creating new applications quickly
```
```suggestion
* Quick start templates that enable users to start creating new applications quickly.
```

ndeevy commented:
Hey @bergerhoffer  I added a couple of tiny suggestions but approved anyway cos it LGTM!
(btw, do you find it useful to use the inline suggested edit feature (as I did here) where you can hit accept and commit the suggestion directly? I like that method but have noticed folks don't use it that much on here :) )

dfitzmau commented:
> Hey @bergerhoffer I added a couple of tiny suggestions but approved anyway cos it LGTM!
> (btw, do you find it useful to use the inline suggested edit feature (as I did here) where you can hit accept and commit the suggestion directly? I like that method but have noticed folks don't use it that much on here :) )

I used the feature twice on GitLab, and the changes I accepted did not get captured in the commit :thermometer: . That was earlier in the year, so maybe the feature is better and possibly more stable on GitHub :thinking: .

bergerhoffer commented:
> Hey @bergerhoffer I added a couple of tiny suggestions but approved anyway cos it LGTM!
> (btw, do you find it useful to use the inline suggested edit feature (as I did here) where you can hit accept and commit the suggestion directly? I like that method but have noticed folks don't use it that much on here :) )

Thanks @ndeevy! I updated per your suggestions.

As for the suggested change feature, I do think that it's really helpful in being able to clearly show where/what your suggestion is, especially if it's complicated. I don't personally use it too much, but I'm used to seeing peer reviewers sometimes use it, but I always just make the change manually. But that's because I'm used to following workflows that only ever having a single commit, so I don't do any changes through the UI.

Anyways, ready for re-review!

inoxx03 commented:
This PR has 3 ACK (including Darragh's). Merging. Will add the usage guidance icon in a separate PR.
